README.md

```diff
@@ -78,6 +78,37 @@ Once connected, run the following make target:
   make local
 ```
 
+### Running Bulk Indexer
+
+#### Locally
+
+Build the bulk indexer by running the following command
+```
+  make reindex
+```
+Then run the executable  
+```
+  ./reindex
+```
+Please make sure your elasticsearch server is running locally on localhost:9092 and version of the server is 7.10, which is the current supported version.
+
+#### Develop
```

Comment:
Probably worth setting the header as `Remote Environment`

And then replace develop with environment or env

Reply:
Done in update 1.